### PR TITLE
Saga tutorial - lesson 1 polishing

### PR DESCRIPTION
**Pull request by mauroservienti**
Started addressing TODOs in lesson 1

### CODE REVIEW

File: tutorials/nservicebus-sagas/1-getting-started/tutorial.md [hunk 1]
```diff
@@ -208,7 +207,7 @@ The snippet above is configuring the endpoint to use `LearningPersistence` desig
 
 #### Running the solution
 
-// TODO: Needs to show some output
+// TODO: Needs to show some output.
```
Comment (mauroservienti) [inline tutorials/nservicebus-sagas/1-getting-started/tutorial.md]: Do we? I mean isn’t is the same exact diagram we have at the beginning?

File: tutorials/nservicebus-sagas/1-getting-started/tutorial.md [hunk 2]
```diff
@@ -220,8 +219,8 @@ You can now press <kbd>F5</kbd> and test the `ShippingPolicy` saga. By sending a
 
 ### Summary
 
-//TODO: Needs a proper summary
+Using NServiceBus Sagas we designed a state machine to satisfy the business requirement that a delivery can be triggered only whne an order is both _accepted_ and _billed_. Sagas are a perfect way to implement business policies, as they describe conditions that needs to be satisfied in order to make a decision. It's generally better to think to sagas as policies then orchestrators or process managers. We want sagas to react to events and, evaluating business rules, make decisions that allow the system to move forward.
 
 ### TODO: Notes
 
-* Consider changing all return Tasks to awaits
+* Consider changing all return Tasks to awaits.
```
Comment (mauroservienti) [inline tutorials/nservicebus-sagas/1-getting-started/tutorial.md]: I’m not anymore sure about this. I’ll review the code first.

**Comment by DavidBoike** [comment]
@mauroservienti I did some editing in the branch.

**Comment by mauroservienti** [comment]
Fixed couple of more things

**Pull request merged**